src/components/calcite-radio-group-item/calcite-radio-group-item.scss

```diff
@@ -1,101 +1,93 @@
 :host {
-  display: flex;
-  align-self: stretch;
-  cursor: pointer;
+  @apply flex self-stretch cursor-pointer;
   transition: background-color 0.1s ease-in-out, border-color 0.1s ease-in-out;
 }
 
 :host label {
-  display: flex;
-  flex: 1;
-  color: var(--calcite-ui-text-3);
+  @apply flex 
+    text-color-3 
+    m-1 
```

Comment:
@bstifle looks like the design needs an update.

Reply:
text-3? for what size

should be

text-1
text--1
text--2

Reply:
It is, with `h` at the end of each.

Reply:
@bstifle I was not seeing the margin in the designs

Reply:
updated the margin to m-px, which is 1px all around like Figma design